### PR TITLE
Paf coverage

### PR DESCRIPTION
**Pull request by yumisims**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/treeval/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/read_coverage.nf [hunk 1]
```diff
@@ -194,18 +144,19 @@ workflow READ_COVERAGE {
     ch_versions             = ch_versions.mix(BEDTOOLS_GENOMECOV.out.versions)
 
     //
-    // MODULE: SORT THE PRIMARY BED FILE
+    // LOGIC: BED2BIGWIG TAKES SORTED COVERAGE BED FILE
```
Comment (weaglesBio) [inline subworkflows/local/read_coverage.nf:147]: Would this be a MODULE rather than LOGIC block?

**Review by weaglesBio — APPROVED** [review]
Only small label change

File: subworkflows/local/read_coverage.nf (hunk 1 above)
Comment (yumisims) [reply subworkflows/local/read_coverage.nf:147]: I think LOGIC, it is a module but to emphasizes why we have to sort it multiple times?

**Comment by yumisims** [comment]
@gq1 @muffato do you thing the missing function/proccess error is a cache problem as I cannot see where is the multiply. Thanks

File: workflows/treeval.nf [hunk 2]
```diff
@@ -256,7 +256,7 @@ workflow TREEVAL {
         REPEAT_DENSITY.out.repeat_density,
         params.entry
     )
-    ch_versions     = ch_versions.mix( HIC_MAPPING.out.versions )
+    ch_versions     = ch_versions.mix( HIC_MAPPING.out.versions )*/
```
Comment (muffato) [inline workflows/treeval.nf:259]: @yumisims . I don't see a matching `/*` for this one. I think it's this `*` that raises the error about "multiply"

Comment (yumisims) [reply workflows/treeval.nf]: ah, thanks

**Pull request merged**